samples/go/photo-dedup/model/commitmeta.go

```diff
@@ -0,0 +1,27 @@
+package model
+
+import (
+	"time"
+
+	"github.com/attic-labs/noms/go/marshal"
+	"github.com/attic-labs/noms/go/types"
+	"github.com/attic-labs/noms/go/d"
+)
+
+type CommitMeta interface {
+	Marshal() types.Struct
+}
+
+type commitMeta struct {
+	Date string
```

Comment:
Other things that would be cool to track:

- distance required by grouping function
- path to input data (ask @willhite about "pinned paths")